packages/bot-web-ui/src/types/stores.types.ts

```diff
@@ -0,0 +1,16 @@
+export type TCoreStore = {
```

Comment:
@shafin-deriv Please use the `TCoreStores` type from `@deriv/stores` package instead 🙏🏻
And if any type is missing please update it in `packages/stores/types.ts` 🙇🏻

Reply:
Got it!! Thanks @farzin-deriv